src/components/Card/Card.tsx

```diff
@@ -26,42 +27,50 @@ const iconMap: { [key: string]: any } = {
 }
 
 const Card = ({plant}: CardProps) => {
-    const [{ isDragging }, dragRef] = useDrag({
+    const [dragging, setIsDragging] = useState<boolean>(false)
+
+    const [{ isDragging }, dragRef, preview] = useDrag({
         type: 'plant',
         item: { plant },
         collect: (monitor) => ({
             isDragging: monitor.isDragging()
         })
     })
 
+    useEffect(() => {
+        setIsDragging(!dragging)
+        // eslint-disable-next-line
+    }, [isDragging])
+
     const draggedCardStyle = isDragging ? {opacity: '.4'} : {}
 
     return (
-        <div className='card' style={draggedCardStyle}>
-            <div className='drag-ref' ref={dragRef}>
+        <>
+            <DragPreviewImage connect={preview} src={`${process.env.PUBLIC_URL}/images/plant.png`} />
```

Comment:
Per Michael's suggestion, change this to `{!isDragging && <DragPreviewImage connect={preview} src={`${process.env.PUBLIC_URL}/images/plant.png`} />}` and it fixed the issue!

Reply:
Line 49 to be exact